apps/src/containedLevels.js

```diff
@@ -74,7 +74,7 @@ export function runAfterPostContainedLevel(fn) {
   if (postState === PostState.None) {
     throw new Error('Shouldnt call runAfterPostContainedLevel before postContainedLevelAttempt');
   }
-  if (PostState.Finished) {
+  if (postState === PostState.Finished) {
```

Comment:
Yes. It doesn't seem to have any user visible impact though, since nothing depends on the result of the post when there's a contained level.